src/components/EntityInfo.vue

```diff
@@ -1,129 +1,138 @@
+              <template v-if="nrNumber">
+                <dt class="mr-2">Name Request No:</dt>
+                <dd id="nr-number">{{ nrNumber }}</dd>
+              </template>
+            </dl>
+          </template>
         </div>
 
-        <div class="business-info__contact">
-          <dl>
-            <dt></dt>
-            <dd id="entity-business-email" aria-label="Business Email">
-              <span>{{businessEmail || 'Unknown Email'}}</span>
-            </dd>
-            <template v-if="fullPhoneNumber">
-              <dt></dt>
-              <dd id="entity-business-phone" aria-label="Business Phone">
-                <span>{{fullPhoneNumber}}</span>
-              </dd>
-            </template>
-          </dl>
-
-          <v-menu bottom left offset-y content-class="v-menu">
-            <template v-slot:activator="{ on }">
-              <v-btn id="entity-settings-button" small icon color="primary" v-on="on">
-                <v-icon small>mdi-cog</v-icon>
+        <div class="right-column text-right align-self-end">
+          <template v-if="businessId">
+            <!-- if LTD or ULC, show _new_ menu -->
+            <div v-if="isLtdOrUlc">
+              <v-btn small text color="primary" id="company-information-button" @click="viewChangeCompanyInfo()">
+                <v-icon small>mdi-pencil</v-icon>
+                <span>View and Change Company Information</span>
               </v-btn>
-            </template>
-            <v-list class="pt-0 pb-0">
-              <v-list-item id="update-business-profile-menuitem" @click="editBusinessProfile()">
-                <v-list-item-title>Update business profile</v-list-item-title>
-              </v-list-item>
-            </v-list>
-          </v-menu>
-        </div>
-      </div>
 
-      <!-- NR Subtitle, NR Number -->
-      <div class="nr-info" v-if="tempRegNumber">
-        <div class="nr-info__meta">
-          <dl>
-            <dt></dt>
-            <dd id="nr-subtitle" aria-label="Sub Title">
-              <span>{{ nrSubtitle }}</span>
-            </dd>
-
-            <template v-if="nrNumber">
-              <dt>Name Request No:</dt>
-              <dd class="ml-2" id="nr-number">
-                <span>{{ nrNumber }}</span>
-              </dd>
-            </template>
-          </dl>
+              <v-menu offset-y>
+                <template v-slot:activator="{ on }">
+                  <!-- FUTURE? -> remove disabled prop -->
+                  <v-btn id="menu-down-button" small icon color="primary" v-on="on" disabled>
+                    <v-icon small>mdi-menu-down</v-icon>
+                  </v-btn>
+                </template>
+
+                <v-list class="pt-0 pb-0">
+                  <v-list-item id="business-summary-menuitem" @click="downloadBusinessSummary()">
+                    <v-icon>mdi-download</v-icon>
+                    <v-list-item-title class="pl-1">Download Business Summary</v-list-item-title>
```

Comment:
This is shown in the uxpin but not described in the ticket, so it's partly done but the v-menu is disabled.